.ci-dockerfiles/llvm-3.9.1/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:14.04
+
+ENV LLVM_VERSION 3.9.1
+
+RUN apt-get update \
+ && apt-get install -y software-properties-common \
+ && add-apt-repository -y ppa:ubuntu-toolchain-r/test \
+ && apt-get update \
+ && apt-get install -y \
+  apt-transport-https \
+  build-essential \
+  g++-6 \
+  git \
+  libncurses5-dev \
+  libssl-dev \
+  make \
+  wget \
+  xz-utils \
+  zlib1g-dev \
+ && wget -O - http://llvm.org/releases/${LLVM_VERSION}/clang+llvm-${LLVM_VERSION}-x86_64-linux-gnu-debian8.tar.xz \
```

Comment:
Change to not use llvm?

Reply:
Not at the moment no. This is how the image was built. We can investigate something later. Also note, once LLVM fixes their issues, this should only be used once in forever if we need to update the image which might be... never.